docs/cli/index.mdx

```diff
@@ -44,6 +107,9 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
 - [temporal task-queue](/cli/task-queue/)
 - [temporal workflow](/cli/workflow/)
 
+
+## Configuration
```

Comment:
I understand why we'd add this heading here, but without something between the h2 and h3 heading it looks off. Is there something we can add here as an intro? If not, feel free to ignore.

Reply:
I agree. I added a very lame sentence but it's something.